tsconfig.json

```diff
@@ -9,9 +16,9 @@
     // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
```

Comment:
Remove the commented-out settings, we can always browse the tsconfig reference for that, no need to have it here as well.